assets/js/vue-apps/components/prompt.vue

```diff
@@ -1,7 +1,7 @@
 <script>
 const { trackEvent } = require('../../helpers/metrics');
 import $ from 'jquery';
-import { includes } from 'lodash';
+import includes from 'lodash/includes';
```

Comment:
This one change allows us to remove the need for `babel-plugin-lodash` which basically just does this.